apps/intellij/src/main/kotlin/dev/nx/console/nxls/WorkspacePathSetting.kt

```diff
@@ -0,0 +1,33 @@
+package dev.nx.console.nxls
+
+import com.intellij.ui.components.JBTextField
+import com.intellij.ui.dsl.builder.MAX_LINE_LENGTH_WORD_WRAP
+import com.intellij.ui.dsl.builder.Panel
+import com.intellij.ui.dsl.gridLayout.HorizontalAlign
+import dev.nx.console.settings.NxConsoleSettingBase
+
+class WorkspacePathSetting : NxConsoleSettingBase<String?> {
+
+    private val inputField = JBTextField()
+    override fun render(panel: Panel) {
+        panel.apply {
+            row {
+                label("Workspace path")
+                cell(inputField)
+                    .horizontalAlign(HorizontalAlign.FILL)
+                    .comment(
+                        "Set this if your Nx workspace is not at the root of the project opened in IntelliJ",
```

Comment:
Dont mention "IntelliJ". Nx Console can be installed on: AppCode, Aqua, CLion, GoLand, IntelliJ, PhpStorm, PyCharm, Rider, RubyMine, AND WebStorm 😅